lib/phubme/slack.ex

```diff
@@ -0,0 +1,25 @@
+defmodule PhubMe.Slack do
+  def send_private_message({:ok, _comment, [], _sender, _source}) do
```

Comment:
Since, the first element of the tuple is always `:ok`, you may not need it. What do you think?